src/LibraryPage.tsx

```diff
@@ -1,9 +1,81 @@
-import React from "react";
-import { useParams } from "react-router-dom";
+import React, { Component } from 'react';
+import { render } from 'react-dom';
+import './style.css';
+import { Card } from "react-bootstrap"
+import {Button} from "react-bootstrap"
 
-const LibraryPage: React.FC = () => {
-  const { id } = useParams();
-  return <span>User ID: {id}</span>;
-};
+class App extends Component {
```

Comment:
Can you turn this into a functional component with a more descriptive name? Check out these if you need some examples, the second shows some with typescript
https://reactjs.org/docs/components-and-props.html
https://levelup.gitconnected.com/using-react-functional-components-with-hooks-in-typescript-6d61075edddc